src/styles/card.scss

```diff
@@ -0,0 +1,152 @@
+@import './variables';
+@import './mixins';
+
+.container {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
+  gap: 48px 48px;
+  justify-content: center;
+  padding: 50px 40px;
+
+  @include on-mobile {
+    grid-template-columns: repeat(2, 200px);
+  }
+
+  @include on-tablet {
+    grid-template-columns: repeat(3, 200px);
+  }
+
+  @include on-desktop {
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+.card {
+  width: 198px;
+  height: 404px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  transition: all 0.3s ease-in-out;
+
+  &__photo {
+    display: flex;
+    justify-content: center;
+    margin: 0;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
+  }
+
+  &__title {
+    color: $main-accent;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: $big-line-height;
+    margin: 40px 16px 0;
+
+    &-content {
+      margin: 0 0 4px;
+    }
+
+    &:hover {
+      color: #34568b;
+    }
+  }
```

Comment:
The card's title color should change when the card is hovered not itself (sorry for overlooking this in my previous review) [Reference](https://github.com/mate-academy/layout_catalog#:~:text=change%20the%20card%20title%20text%20color%20to%20%2334568b%20when%20the%20card%20is%20hovered%20(.card%3Ahover%20.card__title))

Reply:
Oh! I guess I misunderstood the assignment. Thank you tho!